### PR TITLE
redirect app entry to swagger docs and correction on README file

### PR DESCRIPTION
**Pull request by iamGpower**

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -4,12 +4,10 @@ const router = express.Router();
...
 const upload = require('../controllers/upload');
 
-// GET response for '/'
+// GET response for '/' and redirect to documentation page
 router.get('/', (req, res) => {
     
-    res.status(200).json({ 
-        msg: "Welcome to Dockerized User Management App [by TEAM GRANITE]" 
-    });
+    res.status(200).redirect('/api-docs');
```
Comment (goodhands) [inline src/routes/routes.js:10]: Why do we have to redirect it?

Comment (iamGpower) [reply src/routes/routes.js:10]: Not to clutter the entry point with much code or use a separate html doc for that purpose.

Comment (goodhands) [reply src/routes/routes.js:10]: The instruction from Mark is to have the base URL show the swagger doc, we can't redirect it.

**Comment by Kvngston** [comment]
@goodhands you handle this merge or close depending on what you think is right

**Pull request closed without merging**